models/Game.js

```diff
@@ -0,0 +1,75 @@
+const {
+    Model,
+    DataTypes
+  } = require('sequelize');
+  const sequelize = require('../config/connection');
+  // create our Game model
+  class Game extends Model {}
+
+  Game.init({
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true
+    },
+
+    // Need to check the date validator is right
+    date: {
+        type: DataTypes.DATE,
+        allowNull: false,
+        validate: {
+          notNull: {
+            msg: 'A date is needed'
+          },
+          notEmpty: {
+            msg: 'Please provide a date'
+          }
+        }
+      },
+
+      sport_id: {
+        type: DataTypes.INTEGER,
+        references: {
+          model: 'sport',
+          key: 'id'
+        }
+      },
+
+    skill_level: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    equipments_needed: {
+      type: DataTypes.STRING,
```

Comment:
multiple types of equipment might be needed. Chane type to type: DataTypes.ARRAY